code/game/machinery/computer/atmos_alert.dm

```diff
@@ -83,3 +111,8 @@
 		return
 	if(minor_alarms.len)
 		. += "alert:1"
+
+// Subtype which only checks station areas and the mining station
+/obj/machinery/computer/atmos_alert/station_only
+	circuit = /obj/item/circuitboard/computer/atmos_alert/station_only
+	station_only = TRUE
```

Comment:
?

Reply:
Same as above, heavily referenced the tram machines, which had both circuit and machine with the variable